app/models/comment.php

```diff
@@ -42,6 +63,72 @@ public function write()
         );
 
         $db = DB::conn();
-        $db->insert('comment',$params);
+        $db->insert(self::TABLE,$params);
+    }
+
+    public static function getByThreadId($thread_id)
+    {
+        $db = DB::conn();
+        return $db->row(
+            sprintf("SELECT * FROM comment WHERE thread_id = ? LIMIT %d", self::FIRST_COMMENT),
+            array($thread_id)
+        );
+    }
+
+    public static function getIdByThreadId($thread_id)
+    {
+        $db = DB::conn();
+        return $db->value(
+            sprintf("SELECT id FROM comment WHERE thread_id = ? LIMIT %d", self::FIRST_COMMENT),
```

Comment:
Same as this [comment](https://github.com/renzosunico/MyClassroom/pull/2/files#r39232213)
Please apply to all. :bow: